tideways/provision.sh

```diff
@@ -34,6 +34,7 @@ install_tideways_php() {
                 ./configure --enable-tideways-xhprof --with-php-config=php-config$version > /dev/null 2>&1
                 make > /dev/null 2>&1
                 make install > /dev/null 2>&1
+                cd /var/
```

Comment:
Why would it change to `/var`? perhaps `${DIR}` would be better?

Reply:
changed it